documentation/staging/content/userguide/managing-operators/using-helm.md

```diff
@@ -504,6 +505,20 @@ Example:
 elasticSearchPort: 9201
 ```
 
+##### `createLogStashConfigMap`
+Specifies whether a ConfigMap named `weblogic-operator-logstash-cm` should be created during `helm install`. 
+The ConfigMap contains Logstash pipeline configuration for the Logstash container running in the operator pod.
+If set to `true`, a ConfigMap will be created during `helm install` using the `logstash.conf` file in `kubernetes/samples/charts/weblogic-operator` directory.
+Set `createLogStashConfigMap` to `false` if the ConfigMap already exists in the operator's namespace with Logstash configuration provided by the customer.
```

Comment:
with Logstash configuration provided by the customer. -> with the Logstash configuration provided by the user.